src/content/docs/quick-tips/render-elsewhere.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Render views in third party apps
+description: Render Ontime views in third party apps
+---
+
+It is possible to render the any view of Ontime in a third party software that is currently networked with Ontime. \
+This flow would be in any software which can render a HTML page, such as OBS (Open Broadcaster Software), disguise or Touchdesigner.
+
+:::note[Keying views]
+In some views, you will have a key parameter which allows you to configure the background colour.\
+See [view configuration](/quick-tips/configuring-views).
+
+In all other cases you can make use of the [CSS variables](/features/custom-styling) which provide overrides for the background colour:
+
+```css
+:root {
+    --background-color-override: #00FF00;
+}
+```
+:::
+
+## Rendering lower thirds in OBS
+![Lower thirds in OBS](../../../assets/screenshots/features/obs-lower-thirds.png)
+
+* The options **Shutdown source when not visible** and **Refresh browser when scene becomes active** ensure that the lower third animation play on load / unload
```

Comment:
Does any animation actually play on unload? I'd have guessed that shutting down the source just queues up the animation on next load, so
> The options **Shutdown source when not visible** and **Refresh browser when scene becomes active** ensure that the lower third animation plays each time the overlay is reloaded

Reply:
I will need to double-check this. I am not usually an OBS user and this documentation is old.
But yes, the animation should unload and OBS will shutdown the source when the frame is clear